deployment/upgradeLive.sh

```diff
@@ -21,6 +20,7 @@ if [ ! -d "$1" ] ; then
         unlink /opt/kaltura/liveController/latest
         ln -s /opt/kaltura/liveController/$1 /opt/kaltura/liveController/latest
         ln -s /opt/kaltura/liveController/latest/serviceWrappers/linux/kLiveController /etc/init.d/kLiveController
+        bash /opt/kaltura/liveController/$1/deployment/get_bins "$1"
         /etc/init.d/kLiveController restart
         /etc/init.d/recording_uploder.sh restart
         # also better to call dvr saas config
```

Comment:
Ron inserted this (the "eco" print line after) not sure why. Might be irrelevant